recipes/pygmt/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pygmt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d95c2825c3bc368b17e6c5c308179a05f4539aaf2fef4379b6c7bb6125882113
+
+build:
+  number: 0
+  # TODO : pygmt does not yet work with gmt on Windows, see this PR:
+  # https://github.com/GenericMappingTools/pygmt/pull/313. Once that is merged,
+  # the Windows builds here can likely be enabled.
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - gmt >=6.0.0
```

Comment:
Is no upper bound safe here?

Reply:
Do you have a suggestion?

Reply:
Yes it should be safe. PyGMT is a Python wrapper around GMT so we want it to work with any version. That said, we could make it `>=6, <7`. I don't expect GMT7 to be out in a year or two.

Reply:
Happy to change it do what you want @weiji14, I'm not in the position to know what the best call is.

Reply:
Lets put <7 in there, just im case.

Reply:
Done.